docs/admin/user-management.rst

```diff
@@ -18,7 +18,9 @@ superusers that already exist in the CrateDB cluster. The `CREATE USER`_ and
 When CrateDB is started, the cluster contains one predefined superuser. This
 user is called ``crate``. It is not possible to create any other superusers
 
-Users cannot be backed up or restored.
+Users' information is backed up together with the cluster's metadata when a
```

Comment:
```suggestion
User's information is backed up together with the cluster's metadata when a
```

Reply:
Hi Sebastian, it was meant as the information about users plural, but maybe it would be clearer if I reword it to avoid the ' altogether? I could maybe also explain that a restore only creates missing users it does not delete users that did not exist at the time of the backup, what do you think?